src/platform/site-wide/sass/modules/_m-crisis-line.scss

```diff
@@ -1,8 +1,3 @@
-// These should be replaced when the new colors are in the design
```

Comment:
these color vars do exist in the design system https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/52f75afbcd5e4217be513729e50a0fcc0dc92cf2/packages/formation/sass/base/_b-variables.scss#L131-L132